### ISSUE TITLE
Slow and doesn't store audio

### ISSUE THREAD

**Issue by piiskop** [CarnetApp/CarnetNextcloud#116]
**Describe the bug**
The app is so slow that it can't be used for quick notes. I waited and recorded audio which I could play back, however it weren't stored as an error message appeared.
**To Reproduce**
1. Open Carnet from the app menu.
2. Wait... for a minute or so.
3. Click in the right bottom corner on mic.
4. Wait... for a minute or so.
5. Record audio.
6. Play it back.
7. Click OK.
8. Wait... for a minute or so.
9. See the error on the bottom: "Error occured: Uncaught TypeError: Cannot read property 'length' of undefined"

**Expected behavior**
An app for quick notes must be quick as hell. And storing audio must work without errors.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser Chrome
 - Version 18.0.6

**Comment by locness3** [comment]
Possibly related to #24

**Comment by piiskop** [comment]
No, it's not. /24/ is related to UI-design. This issue is about 
non-working storing audio. The next time I tried /Carnet/, it was faster.

*peacecop kalmer:*

*Instituudi tee 3-27 *

*76902 * *Harku *

*Harku*

*Harjumaa*
Estonia

*+372 652 4228 *

*+372  5620 4556 *

*__________________________________________________________________ *

**Comment by locness3** [comment]
Your issue is also about it being slow, and in this way it is related to #24

**Comment by beccon4** [comment]
probably related to #129
